File: crates/aegis-language/tests/corpora/python/perms.py
# perms.py — positive corpus: permission and ownership changes.
#
# Expected: three PermissionOrOwnershipChange operations (os.chmod, os.chown,
# shutil.chown), no modifiers, Known certainty (first positional arg is a string
# literal path). Parses cleanly.
import os
import shutil

os.chmod("/tmp/x", 0o777)
os.chown("/tmp/x", 1000, 1000)
shutil.chown("/tmp/x", "user")

File: crates/aegis-language/tests/corpora/typescript/perms.ts
// Corpus: permission / ownership changes (plan Iteration 7, ADR-022 §3).
//
// `chmodSync` and `chownSync` both fire as `PermissionOrOwnershipChange` with
// default modifiers and `Known` string-literal operands. Mirrors the JavaScript
// corpus's perms file.

fs.chmodSync("x", 0o777);
fs.chownSync("y", 1000, 1000);